ofrak_core/src/ofrak/core/entropy/entropy.py

```diff
@@ -29,7 +29,7 @@ class DataSummary:
     High-level summary of binary data.
 
     :ivar entropy_samples: Shannon entropy of the data. A description of Shannon entropy and how it
-    can be used is [here](../../../../user-guide/key-concepts/gui/minimap.md#entropy-view).
+    can be used is [here](../../../../user-guide/gui/minimap.md#entropy-view).
```

Comment:
``` suggestion
    can be used is [here](../../../../user-guide/key-concepts/gui/minimap.md#entropy-view).
```

Why make this change if the key concepts directory did not change? I don't see this file moved around in this PR, and the old path seems valid?

Reply:
Because the minimap file is actually under GUI, not key concepts.